### PR TITLE
Add Fusion Dependencies

### PR DESCRIPTION
**Pull request by nottirb**
Adds Fusion dependencies to ``wally.toml``.

### CODE REVIEW

File: wally.toml [hunk 1]
```diff
@@ -5,6 +5,9 @@ registry = "https://github.com/UpliftGames/wally-index"
 realm = "shared"
 
 [dependencies]
+Fusion = "nottirb/fusion-beta@0.1.3-beta"
```
Comment (4812571) [inline wally.toml:8]: that is fusion

File: wally.toml [hunk 2]
```diff
@@ -5,6 +5,9 @@ registry = "https://github.com/UpliftGames/wally-index"
...
+FusionUtils = "nottirb/fusion-utils@0.1.4"
```
Comment (4812571) [inline wally.toml:9]: that ... that is fusion utils

File: wally.toml [hunk 3]
```diff
@@ -5,6 +5,9 @@ registry = "https://github.com/UpliftGames/wally-index"
...
+StoryTaskBar = "nottirb/storytaskbar@0.1.4"
```
Comment (4812571) [inline wally.toml:10]: what's a story task bar?

Comment (nottirb) [reply wally.toml:10]: Story taskbar is used to easily create Hoarcekat compatible code with easy-to-use buttons.

Comment (4812571) [reply wally.toml:10]: Neat!

**Review by 4812571 — DISMISSED** [review]
LGTM

**Review by 4812571 — APPROVED** [review]
LGTM!

**Pull request merged**